symphony-bdk-legacy/symphony-api-client-java/src/test/java/it/clients/symphony/api/HealthcheckClientTest.java

```diff
@@ -54,9 +61,16 @@ public void performHealthCheckSuccessWithUnknownField() {
         "\"unknownField\": \"a value\"\r\n" +
         "}");
 
-    HealthcheckResponse response = healthCheckClient.performHealthCheck();
-
+    final HealthcheckResponse response = healthCheckClient.performHealthCheck();
     assertNotNull(response);
-    assertEquals(true, response.getPodConnectivity());
+
+    assertTrue(response.getPodConnectivity());
+    assertTrue(response.getKeyManagerConnectivity());
+    assertTrue(response.getEncryptDecryptSuccess());
+    assertEquals("1.54.1", response.getPodVersion());
+    assertEquals("2.54.0", response.getAgentVersion());
+    assertTrue(response.getAgentServiceUser());
+    assertTrue(response.getCeServiceUser());
+    //assertEquals("a value", response.get);
```

Comment:
Please remove this commented line if useles.

Reply:
Actually I don't know if this line will be usefull or not until someone answer my question about the field **unknownField**

Reply:
@Haiddi-Avencod You can ignore `unknownField`

Reply:
Ok, so should I remove this method `performHealthCheckSuccessWithUnknownField()` ?